schematools/utils.py

```diff
@@ -67,6 +74,25 @@ def defs_from_url(base_url, data_type):
     return schema_lookup
 
 
+def def_from_url(base_url, data_type, dataset_name):
+    """Fetch schema definitions from a remote file for a single dataset
+    The URL could be ``https://schemas.data.amsterdam.nl/datasets/``
+    """
+    schema_lookup = {}
+    if not base_url.endswith("/"):
+        base_url = f"{base_url}/"
```

Comment:
Ik hier ooit een `URL` class voor geschreven. Daarmee kan je heel makelijk delen van een URL aan elkaar concateneren alsmede GET parameters. Het resulterende URL object kan je overal gebruiken waar je string gebruikt.

Zie: https://github.com/workfloworchestrator/nwa-stdlib/blob/5-nwa-stdlib-cache-busting/nwastdlib/url.py (code MIT licensed aldus `setup.py`). Je zou alleen url.py nodig hoeven hebben, niet het hele project)

Reply:
Oh en de bijbehorende tests natuurlijk: https://github.com/workfloworchestrator/nwa-stdlib/blob/5-nwa-stdlib-cache-busting/tests/test_url.py

Reply:
Nice! Altijd elegant, dat overloaden :-)  Zouden we de class kunnen overnemen in utils.py?